Content.Server/Shuttles/ShuttleDestinationSlotComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Server.Shuttles.Components;
+using Content.Shared.Containers.ItemSlots;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Shuttle.Components;
+
+/// <summary>
+/// Enables a shuttle/pod to travel to a destination with an item inserted
+/// </summary>
+[RegisterComponent]
+public sealed partial class ShuttleDestinationSlotComponent : Component
+{
+    [DataField]
+    public ItemSlot DiskSlot = new();
+
+    [DataField]
+    public string DiskSlotId = "Coordinate Disk";
+}
```

Comment:
Putting spaces in yaml keys is a really bad idea and I don't think this is even used.